docs/snippets/vue/component-story-static-asset-with-import.vue3.js.mdx

```diff
@@ -0,0 +1,24 @@
+```js
+// MyComponent.stories.js
+
+import imageFile from './static/image.png';
+
+export default {
+  title: 'img'
+};
+
+const image = {
+  src: imageFile,
+  alt: 'my image'
+};
+
+export const withAnImage = () => {
+  return {
+    setup() {
+       //👇 Returns the content of the image object create above.
+      return { image };
```

Comment:
👍 Cool example!

Reply:
No worries. I was wondering if that was possible to do with Vue 3 when i was testing the examples. I had some concerns...But once i started Storybook it just handled it perfectly :)